vector/src/main/res/layout/item_policy.xml

```diff
@@ -50,16 +48,15 @@
     <ImageView
         android:id="@+id/adapter_item_policy_arrow"
         android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
+        android:layout_height="16dp"
+        android:adjustViewBounds="true"
         android:importantForAccessibility="no"
-        android:paddingStart="8dp"
-        android:paddingEnd="0dp"
         android:rotationY="@integer/rtl_mirror_flip"
         android:src="@drawable/ic_arrow_right"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintTop_toTopOf="parent"
-        app:tint="?vctr_content_secondary"
+        app:tint="?vctr_content_tertiary"
         tools:ignore="MissingPrefix" />
 
 </androidx.constraintlayout.widget.ConstraintLayout>
```

Comment:
Similarly, does our linter not pick up no new line at the end of xml files?

Reply:
I'm not sure the linter checks XML :thinking: